src/commands/Minion/warriorsguild.ts

```diff
@@ -0,0 +1,220 @@
+		const userBank = msg.author.settings.get(UserSettings.Bank);
+
+		// For future
+		/*
+		if (msg.author.skillLevel(SkillsEnum.Attack) !== 99 || msg.author.skillLevel(SkillsEnum.Strength) !== 99 || (msg.author.skillLevel(SkillsEnum.Attack) + msg.author.skillLevel(SkillsEnum.Strength)) < 130) {
+			throw `You need 99 Attack or Strength or a combined attack and strength lvl of 130 to enter the Warriors' guild.`;
+		}
+		*/
+
+		// Temp for now
+		const attack = 99;
+		const strength = 99;
+		if (attack !== 99 || strength !== 99 || attack + strength < 130) {
+			throw `You need 99 Attack or Strength or a combined attack and strength lvl of 130 to enter the Warriors' guild.`;
+		}
+
+		if (minigame === 'animation') {
+			const armour = Armours.find(armour => stringMatches(armour.name, action));
+			if (armour) {
+				const fullhelm = armour.name.concat(' full helm');
+				const platelegs = armour.name.concat(' platelegs');
+				const platebody = armour.name.concat(' platebody');
+				const requiredArmour = resolveItems([fullhelm, platelegs, platebody]);
+				for (const piece of requiredArmour) {
+					if (!bankHasItem(userBank, piece, 1)) {
+						throw `You don't have the required items to kill animated ${action} armour! You are missing at least a ${itemNameFromID(
+							piece
+						)}. A fullhelm, platelegs and platebody are required.`;
+					}
+				}
+
+				// If no quantity provided, set it to the max.
+				if (quantity === null) {
+					quantity = Math.floor(msg.author.maxTripLength / armour.timeToFinish);
+				}
+
+				const duration = armour.timeToFinish * quantity;
+
+				if (duration > msg.author.maxTripLength) {
+					throw `${msg.author.minionName} can't go on trips longer than ${formatDuration(
+						msg.author.maxTripLength
+					)}, try a lower quantity. The highest amount of animated ${
+						armour.name
+					} armour you can kill is ${Math.floor(
+						msg.author.maxTripLength / armour.timeToFinish
+					)}.`;
+				}
+
+				await addSubTaskToActivityTask<AnimatedArmourActivityTaskOptions>(
+					this.client,
+					Tasks.MinigameTicker,
+					{
+						minigameID: MinigameIDsEnum.AnimatedArmour,
+						armourID: armour.name,
+						userID: msg.author.id,
+						channelID: msg.channel.id,
+						quantity,
+						duration,
+						type: Activity.AnimatedArmour
+					}
+				);
+
+				const response = `${msg.author.minionName} is now killing ${quantity}x animated ${
+					armour.name
+				} armour, it'll take around ${formatDuration(duration)} to finish.`;
+
+				return msg.send(response);
+			}
+			throw `That isn't a valid animated armour tier to kill, the available tiers are: ${Armours.map(
+				tier => tier.name
+			).join(', ')}. For example, \`${msg.cmdPrefix}warriorsguild animation 5 bronze\``;
+		}
+
+		if (minigame === 'cyclops') {
+			// Check if either 100 warrior guild tokens or attack cape (similar items in future)
+			if (!bankHasItem(userBank, 8851, 100) && !bankHasItem(userBank, 9747, 1)) {
```

Comment:
This should `if (!bankHasItem(userBank, 8851, 100) || !bankHasItem(userBank, 9747, 1)) {`.